packages/graphql-validate-fixtures/src/validate.ts

```diff
@@ -27,6 +27,51 @@ export interface Error {
   message: string;
 }
 
+export interface GraphQLProjectOperations {
```

Comment:
I think this name is confusing, is this not just `GraphQLProject`?

Reply:
I'll do some renaming to clean up this and the other `FoundOperation`. I agree they are poorly named, I think I was mentally unable to come up with a good name at the time so i just picked something quick to move forward.